src/py42/services/alerts.py

```diff
@@ -134,11 +134,11 @@ def update_note(self, alert_id, note):
         return self._connection.post(uri, json=data)
 
     def get_aggregate_data(self, alert_id):
-        uri = self._uri_prefix.format("query-details-aggregate")
-        data = {
-            "alertId": alert_id,
-        }
-        return self._connection.post(uri, json=data)
+        uri = f"{self._uri_prefix}/v2/query-details-aggregate"
+        data = {"alertId": alert_id}
```

Comment:
One other thing I noticed about this method:  we don't convert the observations to JSON like we do for the `get_details()` method.

Reply:
sad, but it might be too late to do anything different now.

Reply:
If we would have went with the `v2` method approach, I would have done it then.

However, Tim is suggesting that maybe there are 0 actual users of the original method, if that changes anything